BepInEx.Core/Bootstrap/BaseChainloader.cs

```diff
@@ -343,10 +342,13 @@ protected static bool PluginTargetsWrongBepin(PluginInfo pluginInfo)
 		{
 			var pluginTarget = pluginInfo.TargettedBepInExVersion;
 			// X.X.X.x - compare normally. x.x.x.X - nightly build number, ignore
-			if (pluginTarget.Major != CurrentAssemblyVersion.Major) return true;
-			if (pluginTarget.Minor > CurrentAssemblyVersion.Minor) return true;
-			if (pluginTarget.Minor < CurrentAssemblyVersion.Minor) return false;
-			return pluginTarget.Build > CurrentAssemblyVersion.Build;
+			if (pluginTarget.Major != CurrentAssemblyVersion.Major)
+				return true;
+			if (pluginTarget.Minor > CurrentAssemblyVersion.Minor)
+				return true;
+			if (pluginTarget.Minor < CurrentAssemblyVersion.Minor)
+				return false;
+			return pluginTarget.Patch > CurrentAssemblyVersion.Build;
```

Comment:
Formatting this is unnecessary as it makes the diff less clear. We prefer doing formatting in separate commits/PRs so that the relevant diff is properly visible.